src/Pelagos/Bundle/AppBundle/Twig/Extensions.php

```diff
@@ -243,6 +247,24 @@ public function getMaintenanceModeColor() : ? string
         return $this->maintenanceMode->getMaintenanceModeColor();
     }
 
+    /**
+     * Gets maintenance mode color.
+     *
+     * @param string $color The color text.
+     *
+     * @return string|null Returns maintenance mode banner color.
+     */
+    public function maintenanceModeColor(string $color) : string
```

Comment:
We have two functions in this class which gets the maintenance mode color?

Reply:
One of them is a filter, but this one can call the other one.